test/run

```diff
@@ -248,43 +271,69 @@ function run_tests() {
   envs=${PASS:+"-e REDIS_PASSWORD=$PASS"}
   PASS=${PASS:-}
   create_container $name $envs
+  check_result $?
   test_connection "$name" "$PASS"
+  check_result $?
   # Only check version on rhel/centos builds
   if [ "$OS" != "fedora" ]; then
     echo "  Testing scl usage"
     test_scl_usage $name 'redis-server --version' "$VERSION"
+    check_result $?
   fi
   echo "  Testing login accesses"
   local container_ip
   container_ip=$(get_container_ip $name)
   assert_login_access "$container_ip" "$PASS" true
+  check_result $?
   if [ -n "$PASS" ] ; then
     assert_login_access "$container_ip" "${PASS}_foo" false
+    check_result $?
   fi
   assert_local_access "$name"
+  check_result $?
```

Comment:
if function assert_local_access fails "Success" is printed out is it correct?

Reply:
It can be printed. The result is 0. Success means for this test.

Reply:
Definitely not, It has to be fixed.